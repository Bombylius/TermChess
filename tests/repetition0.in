g1 f3 b8 c6 f3 g1 c6 b8 g1 f3 b8 c6 f3 g1 .c c6 b8 .c 
