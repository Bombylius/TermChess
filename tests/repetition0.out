175921860444160
327680
4611686603885117440
43234885634
175921860444160
327680
4611686603885117440
43234885634
rnbqkbnr/pppppppp/8/8/8/8/PPPPPPPP/RNBQKBNR w KQkq - 8 5
4
